validate-swupd.sh

```diff
@@ -131,7 +131,7 @@ for i in $(ls update/www); do
 
 	set -x
 	echo $next | sudo tee $(pwd)/update/www/version/format$3/latest
-	swupd update --path os-$i-install -u file://$(pwd)/update/www -C $(pwd)/Swupd_Root.pem
+	swupd update --path os-$i-install -F $3 -u file://$(pwd)/update/www -C $(pwd)/Swupd_Root.pem
```

Comment:
If you are changing the line already, then please put double quotes around everything that should not be broken apart on whitespace, i.e.

    swupd update --path "os-$i-install" -F "$3" -u "file://$(pwd)/update/www" -C "$(pwd)/Swupd_Root.pem"

Also might be worth using `"$PWD"` rather than `"$(pwd)"` to avoid the need to start up a new shell.

Reply:
Done.